File: 프로그래머스/1/12901. 2016년/2016년.py
def solution(a, b):
    month = [0, 31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
    day = ["FRI", "SAT", "SUN", "MON", "TUE", "WED", "THU"]
    return day[(sum(month[:a]) + b - 1) % 7]

File: 프로그래머스/1/12901. 2016년/2016년.java
class Solution {
    public String solution(int a, int b) {
        int[] month = {0, 31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
        String[] day = {"FRI", "SAT", "SUN", "MON", "TUE", "WED", "THU"};
        
        int sum = 0;
        for (int i = 1; i < a; i++) {
            sum += month[i];
        }
        sum += b;
        return day[(sum - 1) % 7];
    }
}